network_importer/adapters/netbox_api/models.py

```diff
@@ -632,14 +646,19 @@ def update(self, attrs: dict) -> Optional["DiffSyncModel"]:
         """
         nb_params = self.translate_attrs_for_netbox(attrs)
 
-        try:
-            vlan = self.diffsync.netbox.ipam.vlans.get(self.remote_id)
-            clean_params = self.update_clean_tags(nb_params=nb_params, obj=vlan)
-            vlan.update(data=clean_params)
-            LOGGER.info("Updated Vlan %s (%s) in NetBox", self.get_unique_id(), self.remote_id)
-        except pynetbox.core.query.RequestError as exc:
-            LOGGER.warning("Unable to update Vlan %s in %s (%s)", self.get_unique_id(), self.diffsync.name, exc.error)
-            return None
+        if self.remote_id:
```

Comment:
Same here, if we update a vlan it should have a remote_id, so this might be hiding a bigger issue.
Do you know how/when the vlan was created ?